src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOptimizedNullableOperators.cs

```diff
@@ -1109,7 +1109,7 @@ .maxstack  1
                 //         return ((S?)null) < new S?(N4());
                 Diagnostic(ErrorCode.WRN_CmpAlwaysFalse, "((S?)null) < new S?(N4())").WithArguments("S?").WithLocation(41, 16)
                 );
-            var comp = CompileAndVerify(source, expectedOutput: expectedOutput, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular.WithStrictFeature());
+            var comp = CompileAndVerify(source, expectedOutput: expectedOutput, options: TestOptions.ReleaseExe.WithWarningLevel(5));
```

Comment:
I wonder if we should just change many of the `TestOptions` properties to default to using the latest warning level, with the exception of tests specifically designed to ensure that certain warnings do not occur on lower warning levels. (This does not need to happen in this PR.)